pkg/api/norman/customization/vsphere/listers.go

```diff
@@ -295,7 +327,7 @@ func listHosts(ctx context.Context, finder *find.Finder) ([]string, error) {
 		return nil, err
 	}
 
-	data := []string{""} //blank default
+	data := []string{""} // blank default
```

Comment:
TBH I do not know the reason, it has been working this way since the beginning.  I am not aware of any documentation for those endpoints, they are internal endpoints for Rancher UI to fetch data. I think the UI side has handled the case of an empty string well.